tuner/tuner/common.py

```diff
@@ -151,14 +122,21 @@ class Configuration:
     tile_sizes: list[int]
     subgroup_m_count: int
     subgroup_n_count: int
-    gpu_pipeline_options: GpuPipelineOptions
+    gpu_pipeline_options: iree_gpu.PipelineOptionsAttr
     waves_per_eu: int
 
 
 def get_pipeline_config(configuration: Configuration) -> str:
     extra_config = ""
-    if not configuration.gpu_pipeline_options.all_default():
-        extra_config += f", gpu_pipeline_options = {configuration.gpu_pipeline_options}"
+    pipeline_options = configuration.gpu_pipeline_options
+    if (
+        pipeline_options.prefetch_shared_memory is not None
+        or pipeline_options.no_reduce_shared_memory_bank_conflicts is not None
+        or pipeline_options.use_igemm_convolution is not None
+        or pipeline_options.reorder_workgroups_strategy is not None
+    ):
```

Comment:
You can compare `pipeline_options` with a freshly constructed attribute with all default values